### PR TITLE
landing page with info

### PR DESCRIPTION
**Pull request by lundrew**

### CODE REVIEW

File: app/javascript/components/components/Header.js [hunk 1]
```diff
@@ -14,8 +16,9 @@ class Header extends Component {
     console.log("current_user:", current_user)
     return (
       <>
-        <h1>React in Rails with Devise</h1>
+        <h1>nest.</h1>
         <Nav>
+          <NavItem><Link src={Home} className="nav-link">Home</Link></NavItem>
```
Comment (sjproctor) [inline app/javascript/components/components/Header.js:19]: Does this link work? I feel like you would want to use a NavLink with a `to` attribute.

File: app/javascript/components/components/Header.js [hunk 2]
```diff
@@ -14,8 +16,9 @@ class Header extends Component {
     console.log("current_user:", current_user)
     return (
       <>
-        <h1>React in Rails with Devise</h1>
+        <h1>nest.</h1>
```
Comment (sjproctor) [inline app/javascript/components/components/Header.js:17]: Just a reminder that it is not a a good practice to have logs in your main branch.

**Review by sjproctor — APPROVED** [review]
Left some comments but the code I commented on is not included in the current ticket so we can address it when the time comes.

File: app/javascript/components/components/Header.js (hunk 1 above)
Comment (lundrew) [reply app/javascript/components/components/Header.js:19]: have not been able to try, ill be trying it once i create a link to the apartments for next PR

**Pull request merged**